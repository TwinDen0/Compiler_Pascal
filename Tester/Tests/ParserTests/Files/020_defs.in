program CheckDefs;
const
a,b = 5;
begin
;
end.