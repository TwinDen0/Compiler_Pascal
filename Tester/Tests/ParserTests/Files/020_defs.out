(3, 2) Fatal: expected '='
